### ISSUE TITLE
[BUG] Network creation form doesnt react to submit 

### ISSUE THREAD

**Issue by watchoutfreedom** [helpbuttons/helpbuttons#1047]
### 🐞 Describe the Bug

Network creation form doesn't allow to submit in dev and main . 

---

### 🔍 Steps to Reproduce

1. Go to network creation form , complete the form, click submit.

---

### 📚 Additional Context

Add any other context about the problem here:

The pending slogan field : required is preventing the form to submit. Attempt to fix this issue is in the pull request : #1034

**Comment by albjeremias** [comment]
can you please provide the logs of the api?

**Comment by albjeremias** [comment]
most probably is the missing warning to run migrations before proceeding.. https://github.com/helpbuttons/helpbuttons/issues/1046

**Comment by watchoutfreedom** [comment]
> most probably is the missing warning to run migrations before proceeding.. https://github.com/helpbuttons/helpbuttons/issues/1046

Yep Will add the logs later. I think is related to #1034

**Comment by watchoutfreedom** [comment]
Can't take the screencap because the admin is already created and it doesn't allow to login again. 

I can say it was requiring 'slogan' property in the backend  and the form had no field for that so the it was not validating the form.

**Comment by albjeremias** [comment]
sry without the logs of the api, i can't help you.

**Comment by watchoutfreedom** [comment]
These are the logs in the api

GET /api/networks/logo/192 took 0.237 ms
[Nest] 56493  - 01/21/2025, 9:01:05 AM     LOG [HTTP] GET /api/networks/logo/192 404 64B - Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 ::1
GET /api/networks/logo/192 took 0.264 ms
[Nest] 56493  - 01/21/2025, 9:01:05 AM     LOG [HTTP] GET /api/networks/logo/192 404 64B - Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 ::1
POST /networks/new took 191.362 ms
[Nest] 56493  - 01/21/2025, 9:01:07 AM     LOG [HTTP] POST /networks/new 400 143B - Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 ::1
POST /networks/new took 208.391 ms
[Nest] 56493  - 01/21/2025, 8:25:52 PM     LOG [HTTP] POST /networks/new 400 143B - Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 ::1
POST /networks/new took 208.391 ms
[Nest] 56493  - 01/21/2025, 8:25:52 PM     LOG [HTTP] POST /networks/new 400 143B - Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 ::1
no networks found?
[Nest] 56493  - 01/21/2025, 8:27:00 PM   ERROR [Scheduler] HttpException: 🙆🏼‍♂️Default network not found

And these in the nav console

Store.tsx:54 
        
        
       POST http://localhost:3000/api/networks/new 400 (Bad Request)
eval @ ajax.js:191
Observable._trySubscribe @ Observable.js:49
eval @ Observable.js:43
errorContext @ errorContext.js:25
Observable.subscribe @ Observable.js:34
eval @ map.js:12
eval @ lift.js:16
eval @ Observable.js:38
errorContext @ errorContext.js:25
Observable.subscribe @ Observable.js:34
eval @ map.js:12
eval @ lift.js:16
eval @ Observable.js:38
errorContext @ errorContext.js:25
Observable.subscribe @ Observable.js:34
eval @ catchError.js:16
eval @ lift.js:16
eval @ Observable.js:38
errorContext @ errorContext.js:25
Observable.subscribe @ Observable.js:34
_processWatchEvent @ Store.tsx:54
eval @ Store.tsx:36
ConsumerObserver.next @ Subscriber.js:111
Subscriber._next @ Subscriber.js:78
Subscriber.next @ Subscriber.js:49
eval @ withLatestFrom.js:45
OperatorSubscriber._this._next @ OperatorSubscriber.js:22
Subscriber.next @ Subscriber.js:49
eval @ filter.js:12
OperatorSubscriber._this._next @ OperatorSubscriber.js:22
Subscriber.next @ Subscriber.js:49
eval @ Subject.js:52
errorContext @ errorContext.js:25
Subject.next @ Subject.js:42
next @ share.js:56
ConsumerObserver.next @ Subscriber.js:111
Subscriber._next @ Subscriber.js:78
Subscriber.next @ Subscriber.js:49
eval @ Subject.js:52
errorContext @ errorContext.js:25
Subject.next @ Subject.js:42
AnonymousSubject.next @ Subject.js:156
emit @ Store.tsx:20
onSubmit @ index.tsx:57
eval @ index.esm.mjs:1974
await in eval
callCallback @ react-dom.development.js:4164
invokeGuardedCallbackDev @ react-dom.development.js:4213
invokeGuardedCallback @ react-dom.development.js:4277
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:4291
executeDispatch @ react-dom.development.js:9041
processDispatchQueueItemsInOrder @ react-dom.development.js:9073
processDispatchQueue @ react-dom.development.js:9086
dispatchEventsForPlugins @ react-dom.development.js:9097
eval @ react-dom.development.js:9288
batchedUpdates$1 @ react-dom.development.js:26135
batchedUpdates @ react-dom.development.js:3991
dispatchEventForPluginEventSystem @ react-dom.development.js:9287
dispatchEventWithEnableCapturePhaseSelectiveHydrationWithoutDiscreteEventReplay @ react-dom.development.js:6465
dispatchEvent @ react-dom.development.js:6457
dispatchDiscreteEvent @ react-dom.development.js:6430Understand this errorAI
index.ts:37 Validation errors {"statusCode":400,"success":false,"message":"validation-error","validationErrors":{"slogan":["slogan is too short","slogan must be a string"]}}

Cheers

**Comment by watchoutfreedom** [comment]
did you see https://github.com/helpbuttons/helpbuttons/pull/1034  ?

**Comment by albjeremias** [comment]
its clear, you are requesting the slogan on the api, and you are not requesting it on the frontend.
